pkg/types/types.go

```diff
@@ -134,11 +134,13 @@ const (
 type RunSpec struct {
 	DeviceMappings []DeviceMapping `json:"DeviceMappings"` //required for all compilers
 	// DefaultInstanceMemory is in MB
-	DefaultInstanceMemory int                `json:"DefaultInstanceMemory"` //required for all compilers
-	MinInstanceDiskMB     int                `json:"MinInstanceDiskMB"`
-	StorageDriver         StorageDriver      `json:"StorageDriver,omitempty"`
-	VsphereNetworkType    VsphereNetworkType `json:"VsphereNetworkType"`
-	Compiler              string             `json:"Compiler,omitempty"`
+	DefaultInstanceMemory int `json:"DefaultInstanceMemory"` //required for all compilers
```

Comment:
Can you shorten the comment here? I really don't like the fact that fields are misaligned just because of the comment.